src/service/fixed-layer.js

```diff
@@ -160,6 +158,11 @@ export class FixedLayer {
    * Must be always called after DOMReady.
    */
   setup() {
+    const isEmbedded = !!this.ampdoc.getParent();
```

Comment:
Are you looking for "isIframed" here? If so, I don't think `!!ampdoc.getParent()` would give us this, since we do not consider iframe to be a case of a chain of ampdocs where runtime might be reused. If so, the signal we need is in the `viewer.isEmbedded()`. Notice that that signal is needed for both: iframed and in-a-webivew cases since both can offset header. Another recommendation here: to also switch on this mode in `localDev` mode to make local development/testing easier.

Reply:
That's exactly what I was looking for Dima, thank you!

Reply:
Moved to `Viewer.isEmbedded` and updated the test suite to match.